test/platform/gitlab/index.spec.ts

```diff
@@ -447,9 +447,9 @@ describe('platform/gitlab', () => {
     it('returns status if name found', async () => {
       api.get.mockReturnValueOnce({
         body: [
-          { name: 'context-1', state: 'pending' },
-          { name: 'some-context', state: 'success' },
-          { name: 'context-3', state: 'failed' },
+          { name: 'context-1', status: 'pending' },
```

Comment:
Have you checkd the gitlab api for correct property name?

Reply:
Yeah: https://docs.gitlab.com/ee/api/commits.html#list-the-statuses-of-a-commit
Also, correct property name used in other test (line 439)